src/NosCore.Data/WebApi/PostedPacket.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NosCore.Shared.Enumerations;
+
+namespace NosCore.Data.WebApi
+{
+    public class PostedPacket
+    {
+	    public string Packet { get; set; }
+
+	    public string SenderCharacterName { get; set; }
+
+	    public string ReceiverCharacterName { get; set; }
+
+	    public long SenderCharacterId { get; set; }
```

Comment:
Create a new object name CharacterData in webApi and put CharacterName and CharacterId into it